stock_analytic/models/analytic_applicability.py

```diff
@@ -11,3 +11,21 @@ class AccountAnalyticApplicability(models.Model):
         selection_add=[("stock_move", "Stock Move")],
         ondelete={"stock_move": "cascade"},
     )
+    stock_picking_type_id = fields.Many2one(
+        "stock.picking.type",
+        string="Operation Type",
+    )
+
+    def _get_score(self, **kwargs):
+        score = super()._get_score(**kwargs)
+        if score == -1:
+            return -1
+        picking_type = self.env["stock.picking.type"].browse(
+            kwargs.get("picking_type", None)
+        )
+        if picking_type and self.stock_picking_type_id:
+            if picking_type == self.stock_picking_type_id:
```

Comment:
To make it more concise.

```suggestion
        if score >= 0 and self.stock_picking_type_id:
            if kwargs.get("picking_type") == self.stock_picking_type_id.id:
```